project_task_employee/models/project.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    employee_ids = fields.Many2many(
```

Comment:
Shouldn't we link to `res.users` here instead? Since https://github.com/odoo/odoo/blob/7e5d5045bfa7752176dd8f19c546ff6b19841381/addons/project/models/project.py#L439